dkpro-core-io-nyt-asl/src/main/resources/desc/type/NYTArticleMetaData.xml

```diff
@@ -0,0 +1,239 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<typeSystemDescription xmlns="http://uima.apache.org/resourceSpecifier">
+	<name>NYTArticleMetaData</name>
+	<description />
+	<version>1.0</version>
+	<vendor />
+	<types>
+		<typeDescription>
+			<name>de.tudarmstadt.ukp.dkpro.io.nyt.metadata.NYTArticleMetaData
+			</name>
```

Comment:
In new modules, the package names should be `org.dkpro.core...` instead of `de.tudarmstadt.ukp.dkpro.core.`. In any case, there is a "core" missing here ;)